modules/reference/pages/feature/appSecurity/examples.adoc

```diff
@@ -53,7 +53,7 @@ LTPA configuration can provide Single Sign-on (SSO) to secure applications. For
 
 You can configure an authentication filter to secure requests for accessing protected resources that are used with LTPA for authentication.
 The LTPA supports authentication filters.
-For more information on configuring an authentication filter, see xref:ROOT:authentication-filters.adoc[Authentication filters].
+For more information on configuring an authentication filter, see xref:ROOT:authentication-filters.adoc[Authentication filters]. 
```

Comment:
For more information, see xref:ROOT:authentication-filters.adoc[Authentication filters